pkg/trace/writer/stats.go

```diff
@@ -253,10 +253,16 @@ func splitPayload(p pb.ClientStatsPayload, maxEntriesPerPayload int) []clientSta
 		payloads[i] = clientStatsPayload{
 			bucketIndexes: make(map[timeWindow]int, 1),
 			ClientStatsPayload: pb.ClientStatsPayload{
-				Hostname: p.Hostname,
-				Env:      p.Env,
-				Version:  p.Version,
-				Stats:    make([]pb.ClientStatsBucket, 0, maxEntriesPerPayload),
+				Hostname:         p.Hostname,
+				Env:              p.Env,
+				Version:          p.Version,
+				Service:          p.Service,
+				Lang:             p.Lang,
+				TracerVersion:    p.TracerVersion,
+				RuntimeID:        p.RuntimeID,
+				Sequence:         p.Sequence,
+				AgentAggregation: p.AgentAggregation,
```

Comment:
I think `RuntimeID` and `AgentAggregation` were in the protobuf before? Why didn't we set them previously?

Reply:
It was an error not to set them previously.